features/step_definitions/salt_steps.rb

```diff
@@ -446,23 +446,42 @@
   $minion.run(cmd, false)
 end
 
-Then(/^I click on the css "(.*)" until page does not contain "([^"]*)" text$/) do |css, arg1|
+Then(/^I click on the css "(.*)" until page does not contain "([^"]*)" text$/) do |css, text|
   not_found = false
   begin
-    Timeout.timeout(30) do
+    Timeout.timeout(DEFAULT_TIMEOUT) do
       loop do
-        unless page.has_content?(arg1)
+        unless page.has_content?(text)
           not_found = true
           break
         end
         find(css).click
       end
     end
   rescue Timeout::Error
-    raise "'#{arg1}' cannot be found after several tries"
+    raise "'#{text}' still found after several tries"
```

Comment:
Not in this case. This particular step is actually failing if `#{text}` is still found after the timeout is reached.

The case you're mentioning is just covered by the next step implemented on this file: https://github.com/SUSE/spacewalk-testsuite-base/pull/387/files#diff-daf5562d26751e108a56ac4ad5257bc9R480